aiohttp/client_reqrep.py

```diff
@@ -475,7 +477,10 @@ def keep_alive(self):
             if self.url.raw_query_string:
                 path += '?' + self.url.raw_query_string
 
-        writer = StreamWriter(conn.protocol, conn.transport, self.loop)
+        writer = StreamWriter(
+            conn.protocol, conn.transport, self.loop,
+            on_chunk_sent=self._on_chunk_request_sent
```

Comment:
`StreamWriter` is used in (at least) two scenarios:
- here, for writing request body
- in RequestHandler for writing response to incoming request

In the later case, having `traces` an calling `on_request_chunk_sent` seems wrong, because the class is used for writting something completely different. So `on_chunk_sent` is introduced to have something generic.

Reply:
Oks good point!

Reply:
+1